travis/.travis.yml

```diff
@@ -6,7 +6,8 @@ cache:
 before_install:
   - "script/clone_all_rspec_repos"
   # Note this doesn't work on JRUBY 2.0.0 mode so we don't do it, the excluded versions are broken on Ruby 2.3
-  - if [ "jruby" != "$TRAVIS_RUBY_VERSION"  ]; then gem install bundler --version "!= 1.12.0, != 1.12.1"; fi
+  - if [ "jruby" != "$TRAVIS_RUBY_VERSION"  ]; then gem install bundler --version "!= 1.12.0, != 1.12.1, != 1.12.2, != 1.12.3"; fi
+install: if [ "jruby" != "$TRAVIS_RUBY_VERSION"  ]; then bundle _1.11.2_ install --binstubs --standalone --without documentation --path ../bundle; else bundle install --binstubs --standalone --without documentation --path ../bundle; fi
```

Comment:
Honestly, it seems simpler to just install 1.11 (and specify just that version) util there's a new release that works.